collections.go

```diff
@@ -25,6 +25,10 @@ func (cl *CollectionsClient) List() *CollectionsListClient {
 	return newCollectionListClient(cl.sl)
 }
 
+func (cl *CollectionsClient) Create(name CollectionName) *CollectionsCreateClient {
```

Comment:
No need to create new type here. Types like `CollectionID` represent something unique hence to differentiate them from regular strings we create new type aliases. But here  the `name` has no other property to separate it from a regular string.